assets/js/components/notifications/FirstPartyModeWarningNotification.js

```diff
@@ -73,14 +74,9 @@ export default function FirstPartyModeWarningNotification( {
 						),
 					}
 				) }
-				dismissCTA={
-					<Dismiss
-						id={ id }
-						primary={ false }
-						dismissLabel={ __( 'Got it', 'google-site-kit' ) }
-					/>
-				}
-				type="warning"
+				dismissButton={ {
```

Comment:
I've also allowed `dismissButton` to be a boolean in the `NoticeNotification` component to avoid passing in labels unnecessarily. You can cherry pick this commit as well: https://github.com/google/site-kit-wp/pull/10786/commits/4d5a9fe27264308edcac55874a593c431500ff61